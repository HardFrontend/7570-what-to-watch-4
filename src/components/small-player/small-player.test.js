import React from "react";
import renderer from "react-test-renderer";
import SmallPlayer from "./small-player.jsx";

const film =
  {
    id: 0,
    title: `Fantastic Beasts`,
    img: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    genre: `comedy`,
    year: `2011`,
    videosrc: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
  };


it(`VideoPlayer render`, () => {
  const tree = renderer
    .create(<SmallPlayer
      poster={film.img}
      src={film.videosrc}
      isMuted={true}
      isPlaying={true}
      width="280" height="175"
    />, {
      createNodeMock: () => {
        return {};
      }
    }
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
