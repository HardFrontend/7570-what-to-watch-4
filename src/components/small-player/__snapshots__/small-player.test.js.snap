// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer render 1`] = `null`;
